examples/image-resize/app/root.tsx

```diff
@@ -0,0 +1,27 @@
+import { Links, LiveReload, Meta, Outlet, Scripts, ScrollRestoration } from 'remix'
+import type { MetaFunction } from 'remix'
+
+export const meta: MetaFunction = () => {
+  return { title: 'New Remix App' }
+}
+
+export default function App() {
+  return (
+    <html lang="en">
+      <head>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        {/* only for demo purposes, do not use tailwind play cdn for your live website */}
+        <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
We don't need the styling to be nice in these examples. Could you remove this (and tailwind classes)? (Keep in mind that most people will just look at the code anyway).

```suggestion
```

Reply:
Sure, I can remove it.
it was only for testing purposes